pytest_splinter/plugin.py

```diff
@@ -321,14 +325,19 @@ def make_screenshot_on_failure():
                 else:
                     screenshot_dir = tmpdir.mkdir('screenshots').strpath
                 screenshot_path = os.path.join(screenshot_dir, screenshot_file_name)
-                browser.driver.save_screenshot(screenshot_path)
-                with open(screenshot_path) as fd:
-                    if slaveoutput is not None:
-                        slaveoutput.setdefault('screenshots', []).append({
-                            'class_name': classname,
-                            'file_name': screenshot_file_name,
-                            'content': fd.read()
-                        })
+                try:
+                    LOGGER.info('Saving screenshot to %s' % (screenshot_path,))
+                    browser.driver.save_screenshot(screenshot_path)
+                    with open(screenshot_path) as fd:
+                        if slaveoutput is not None:
+                            slaveoutput.setdefault('screenshots', []).append({
+                                'class_name': classname,
+                                'file_name': screenshot_file_name,
+                                'content': fd.read()
+                            })
+                except URLError as e:
+                    LOGGER.warn('Could not save screenshot: %s' % (e,))
```

Comment:
request.config.warn('splinter', "Could not save screenshot: {0}".format(e))
this way it will be visible if you pass -r to pytest (also would be nice to mention that in readme)